presto-product-tests/src/main/java/com/facebook/presto/tests/convention/TestShowPartitions.java

```diff
@@ -37,20 +43,30 @@
 import static com.teradata.tempto.fulfillment.table.hive.InlineDataSource.createResourceDataSource;
 import static com.teradata.tempto.fulfillment.table.hive.InlineDataSource.createStringDataSource;
 import static com.teradata.tempto.query.QueryExecutor.query;
+import static java.lang.Math.min;
+import static java.lang.String.format;
+import static java.util.Objects.requireNonNull;
+import static org.testng.Assert.assertEquals;
 
 public class TestShowPartitions
         extends ProductTest
         implements RequirementsProvider
 {
+    // We use hive.max-partitions-per-scan=100 in tests, so this many partitions is too many
+    private static final int TOO_MANY_PARTITIONS = 105;
```

Comment:
maybe use constant for maximum allowed (100)?

Reply:
What's the difference?

Reply:
Becase 105 is not more special than 106 or 104, while 100 is magic. It points to a limit, so it has more information in it.

Reply:
This is why there is a comment explaining what this number is. I cannot define `MAX_PARTITIONS_PER_SCAN=100` constant here, because it might be a lie (if tests configuration changes). This is why i decided to go with constant that has less specific meaning, and backed it up with verification in the test, that we indeed are above the limit.